routes/celebrities.js

```diff
@@ -0,0 +1,58 @@
+const Celebrity = require("../models/celebrity");
+
+const router = require("express").Router();
+
+router.get('/', (req, res, next) =>{ //browser require the root
+    Celebrity.find()
+    .then(celebrities => { //whatever come back has this name
+        console.log(celebrities)
+        res.render('celebrities/index', {celebrities}); // celebrities is what's the API return to me - could be any name
+        //always make sure to pass an object in render as second parameter
+})
+.catch(err => {
+    next(err);
+})
+})
+
+
+router.get('/new', (req,res, next) => {
+    console.log('Something')
+    res.render('celebrities/new');
+})
+
+router.get('/:id/delete', (req,res, next) => {
+    console.log('test')
+    Celebrity.findByIdAndRemove(req.params.id)//{ _id: req.params.id } 
+    .then(() =>{
+    res.redirect('/celebrities')
+})
+    .catch(err => {
+    next(err);
+})
+})
```

Comment:
This works they way it is but I think the lab requires this to be a post instead. This means you would need to use a form to delete in the hbs file.